onnxruntime/python/tools/transformers/models/whisper/whisper_helper.py

```diff
@@ -354,7 +406,7 @@ def verify_onnx(
         diff = pt_outputs - ort_outputs
         max_diff = max(diff.min(), diff.max(), key=abs)
 
-        if max_diff > 0:
+        if max_diff >= 0:
```

Comment:
This may cause issues with verifying the transcriptions because sometimes there is a comma and sometimes there isn't. Can we revert this for now? I have a PR that rewrites the parity checking to handle these cases.